packages/core/src/tree-view/tree.stories.ts

```diff
@@ -1,14 +1,18 @@
 /*
- * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import '@cds/core/alert/register.js';
+import '@cds/core/button/register.js';
 import '@cds/core/icon/register.js';
 import '@cds/core/tree-view/register.js';
 import { CdsTreeItem } from '@cds/core/tree-view';
 
-import { html } from 'lit';
+import { css, html, LitElement } from 'lit';
+import { createItems, TreeItem } from './utils';
```

Comment:
Path should be have `.js` extension`./utils.js`. It can get a bit buggy if storybook is using import maps without it